resources/lib/addon_utils.py

```diff
@@ -130,9 +130,8 @@ def clear_cache(plugin):
     urlquick.cache_cleanup(-1)
     Script.notify(plugin.localize(30371), '')
 
-    # Remove all tv guides
     dirs, files = xbmcvfs.listdir(Script.get_info('profile'))
     for fn in files:
-        if '.xml' in fn and fn != 'settings.xml':
-            Script.log('Remove xmltv file: {}'.format(fn))
+        if ('.xml' in fn and fn != 'settings.xml') or '.json' in fn:
```

Comment:
```suggestion
        if (fn.endswith('.xml') and fn != 'settings.xml') or fn.endswith('.json'):
```
This is more correct